src/bp-members/bp-members-functions.php

```diff
@@ -5445,14 +5475,29 @@ function bb_core_get_user_slug( int $user_id ) {
  *
  * @since BuddyBoss 2.3.1
  *
```

Comment:
```suggestion
 * @since BuddyBoss [BBVERSION] The `$force` parameter was added.
```

Enhancement - We should start adding multiple since tags if we make any changes to existing functions.